02-basics-2/20-broken-map/__tests__/broken-map.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, it, expect } from 'vitest'
+import { mount } from '@vue/test-utils'
+import MapApp from '../MapApp.js'
```

Comment:
After renaming the module

```suggestion
import MapApp from '../MapApp.ts'
```